src/snapshot/svgtoimg.js

```diff
@@ -9,62 +9,61 @@
 
 'use strict';
 
-var EventEmitter = require('events').EventEmitter;
-
 function svgToImg(opts) {
```

Comment:
I think the solution would be to make `svgToImg` return an event object by default and optionally an promise (e.g. in the case of `Plotly.Snapshot`).